src/GeositeFramework/js/Plugin.js

```diff
@@ -97,6 +99,28 @@ require(['use!Geosite',
             }
         }
 
+        function toggleMobileMap() {
+            if (!this.singlePluginMode) {
+                console.error('Programmatic map toggle only available in single plugin mode');
```

Comment:
This is just my preference, but I don't feel that we need to add a message to the console is these cases. What do you think @mmcfarland?

Reply:
Going to drop these error messages; without them, these functions are small enough that we can just make the assignments inline in the initializer:

```js
showMobileMap = (N.app.singlePluginMode && N.app.showMobileMap) ?
    N.app.showMobileMap : _.noop(),
showMobileContent = (N.app.singlePluginMode && N.app.showMobileContent) ?
    N.app.showMobileContent : _.noop();
```